ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/SingleEntityVulnerabilitiesTable.tsx

```diff
@@ -81,7 +85,7 @@ function SingleEntityVulnerabilitiesTable({
                         CVE Status
                         {isFiltered && <DynamicColumnIcon />}
                     </Th>
-                    {/* TODO Add sorting for these columns once aggregate sorting is available in BE */}
+                    <Th sort={getSortParams('CVSS')}>CVSS</Th>
```

Comment:
just double checking, will this be sortable? it seems like a lot of the sort functions outside of the basic resources are limited by performance.

Reply:
Yeah, I added this to the doc and commented on it and Mandar approved it as sortable. The `topCvss` score on the main overview page is _not_ sortable though. I guess `imageCve->topCvss` is a derived field while `image->imageVulnerabilities->cvss` is not, which makes a difference for sorting performance.

IIRC, derived fields need to load the entire data set into memory, sort it, and then return a page of data, while non-derived fields pull a single sorted page directly from the DB.